modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/SelectMLSS.pm

```diff
@@ -207,6 +207,13 @@ sub _map_mlss_to_db {
 
 	# check all have been mapped
 	foreach my $this_mlss_id ( @$mlsses ) {
+        my $mlss_adap = $self->compara_dba->get_MethodLinkSpeciesSetAdaptor;
+        unless ( $mlss_db_mapping{$this_mlss_id} ) {
+            my $this_mlss = $mlss_adap->fetch_by_dbID($this_mlss_id);
+            if ( $this_mlss ) {
+                $mlss_db_mapping{$this_mlss->dbID} = $self->param('master_db');
```

Comment:
Isn't the `$mlss_db_mapping` supposed to point mlsses to the location of their alignments? Why would we use master as a default - there are never any alignments stored in master.

Reply:
This was your addition when you were helping me :wink: 
I am in the process of pruning changes because I misunderstood the task :+1: